app/models.py

```diff
@@ -0,0 +1,91 @@
+import pytz
+import uuid
+
+from django.db import models
+
+# Create your models here.
+
+class User(models.Model):
+    mentor = models.BooleanField(default=False)
+    mentee = models.BooleanField(default=True)
+    user_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    username = models.CharField(blank=False, max_length=200)
+    about = models.TextField(max_length=1000, help_text='a brief description of you')
+    avatar = models.ImageField(upload_to='images/')
+    stacks = models.ForeignKey('Stack', on_delete=models.CASCADE)
+
+    PRONOUN_CHOICES = [
+        ('They', 'They/Them'),
+        ('She', 'She/Her'),
+        ('He', 'He/Him')
+    ]
+    preferred_pronouns = models.CharField(
+        max_length=5,
+        choices=PRONOUN_CHOICES,
+        null=False
+    )
+    spoken_languages = models.OneToOneField('Spoken_Languages', on_delete=models.SET_DEFAULT, default='English')
+    blog_link = models.URLField(max_length=200)
+
+    ALL_TIMEZONES = sorted((item, item) for item in pytz.all_timezones)
```

Comment:
Please provide the package's readme as a comment here just for ease of readability